octobot_trading/exchanges/types/websocket_exchange.py

```diff
@@ -178,7 +178,7 @@ def on_pong(self):
 
     async def on_ping(self):
         self.logger.debug("Ping received. Sending pong...")
-        self.websocket.pong()
+        await self.websocket.pong()
```

Comment:
?

Reply:
Ping and pong methods are async https://websockets.readthedocs.io/en/stable/api.html#websockets.protocol.WebSocketCommonProtocol.ping

Reply:
Yes my bad 👍